### ISSUE TITLE
"Options" button for Requests not working; dim screen

### ISSUE THREAD

**Issue by supermarkert** [Ombi-app/Ombi#4208]
**Describe the bug**
For an active Request, it shows 2 buttons: "Details" and a red "Options" button. Details works fine.
Clicking "Options" causes the web/screen to dim slightly, as if a window is opening, however nothing appears except a small tab at the bottom of the screen starts to appear but never does.  Pressing "Esc" clears the shadow window and I can return to the Requests screen.

**To Reproduce**
1. Have an active Request available in the Requests section.
2. Click on the red "Options" button.

**Expected behavior**
I expect the Options to appear, but they don't.

**Screenshots**
Before:
![image](https://user-images.githubusercontent.com/15205204/119364474-f22e0180-bc7c-11eb-8b98-259198067fb5.png)

Clicked:
![image](https://user-images.githubusercontent.com/15205204/119364537-007c1d80-bc7d-11eb-964e-6b39046fc49b.png)

**Logs (Logs directory where Ombi is located)**
(not available at time of issue)

**Desktop (please complete the following information):**
 - OS: Windows 10

**Ombi Version (please complete the following information):**
 - Version [4.0.1394] 
- Media Server [Plex]
- Database Type: SQLite

**Additional context**
(Note: relatively new install, so not sure if it was ever working previously)

**Comment by tidusjar** [comment]
Are there any errors in your browser console? F12.

If so can you press the network tab and then refresh the page and you should see something like
![image](https://cloud.githubusercontent.com/assets/6642220/23088047/eac44abc-f570-11e6-9f23-d2eb022ed059.png)


If you click the red text and then the preview tab like the image above and post the result?

**Comment by supermarkert** [comment]
Starts with these 2 errors:
![image](https://user-images.githubusercontent.com/15205204/119369914-b302af00-bc82-11eb-82bd-1fdbd1453a51.png)

Then numerous errors pour in after:
![image](https://user-images.githubusercontent.com/15205204/119369982-c44bbb80-bc82-11eb-850d-d35ab7c807fe.png)

**Comment by supermarkert** [comment]
Upon further review, this is what I get when clicking either Red error ("No connection with that ID"):
![image](https://user-images.githubusercontent.com/15205204/119374865-1cd18780-bc88-11eb-958a-3fb6266fc7b7.png)

**Comment by chiefo0306** [comment]
I saw supermarket mentioned it was a relatively new install so he wasn't sure if it previously worked. I've been running V4 beta for some months now. Looks like this is relatively new. I'm seeing the issue in both chrome on desktop and chrome on android

**Comment by tidusjar** [comment]
This is a new issue, plan to fix it asap.

**Comment by chiefo0306** [comment]
Some more information. I opened the debug panel and stretched it out. After refreshing the page, with ombi only using rough 1/4 of the screen and the browser debug on 3/4, it works 

![image](https://user-images.githubusercontent.com/51934389/119571582-22f26180-bd77-11eb-9d82-e26ad24cfd29.png)


So it seems it is rendering. Now that I fixed it by expanding the debug window, it works fine even with debug removed and refreshing the page:

![image](https://user-images.githubusercontent.com/51934389/119571775-65b43980-bd77-11eb-9a80-81117ea71f00.png)
